lib/extensions/simple_form/components/html5_no_aria_required.rb

```diff
@@ -0,0 +1,15 @@
+module SimpleForm
+  module Components
+    module HTML5NoAriaRequired
+      include HTML5
```

Comment:
I was thinking this include meant we're extending the original module. Wouldn't that cover those other inputs? @aduth

Reply:
That's a good point, I forgot about that. But I did test this locally and it did regress.

What I think it might be is that maybe the constructor isn't being initialized anymore because we're `use :html5_no_aria_required` and not `use :html5` ?

You can try it out:

```
diff --git a/app/views/devise/sessions/new.html.erb b/app/views/devise/sessions/new.html.erb
index 784af585e..20352af39 100644
--- a/app/views/devise/sessions/new.html.erb
+++ b/app/views/devise/sessions/new.html.erb
@@ -30,6 +30,7 @@
       html: { autocomplete: 'off' },
     ) do |f|
 %>
+  <%= f.input(:email, as: :numeric) %>
   <%= render ValidatedFieldComponent.new(
         form: f,
         name: :email,
```